src/chexus/validators.py

```diff
@@ -211,6 +211,64 @@ def validate(self, node: Dataset | Group) -> Violation | None:
             return Violation(node.name)
 
 
+class transformation_offset_units_invalid(Validator):
+    def __init__(self) -> None:
+        super().__init__(
+            "transformation_offset_units_invalid",
+            "Transformation offset_units attr. should be a length unit",
+        )
+
+    def applies_to(self, node: Dataset | Group) -> bool:
+        return (
+            isinstance(node, Dataset)
```

Comment:
Even if the transform is given by an NXlog it can have `offset_units`, should not exclude `Group`?

Reply:
Makes sense, should we include `Group` here too? https://github.com/scipp/chexus/pull/41/files/259702786e8501845328afb2d625afc9433e4850#diff-3c9868e9a791bf461c0b2185174fea8b7ede108971ed45b3aee30b66d3be87bdR195-R211

Reply:
Yes, we should!